scenario1/experiment_orchestrator.py

```diff
@@ -268,13 +295,14 @@ def run_experiment(model, mode, dir_name: str):
     print(f"{'='*50}")
 
     benchmark_name = "scenario1"
+    exp_folder = f"{time.strftime("%Y%m%d-%H%M%S")}_{benchmark_name}"
 
     # Start server
-    pod_name = start_vllm_server_client(full_config, benchmark_name, core_v1, mode, model)
+    pod_name = start_vllm_server_client(full_config, exp_folder, core_v1, mode, model)
     print(f"Created pod '{pod_name}'")
 
     while True:
-        remote_file_path = f"/vllm-data-collection/scenario1/scenario1_output_{mode}.json"
+        remote_file_path = f"/mnt/{exp_folder}/results/scenario1_output_{mode}.json"
```

Comment:
I would say `/mnt/results/{model}/{exp_folder}/scenario1_output_{mode}.json`, WDYT? Put all results under `/mnt/results` and grouped by model name.